tests/end_to_end/http_server/large_files/test/auto_tests.ts

```diff
@@ -10,22 +10,28 @@ const autoGenAutoUploadSmallFileInfos: [number, Unit][] = [
     // Edge Cases
     [0, 'B'],
     [1, 'B'],
-    [120 * 1024 * 1024 + 1, 'B'], // One more byte than can be processed in a single hash_file_by_parts call
-    [2_000_000 + 1, 'B'], // One more byte that the message chunk size
 
     // General Cases
     [1, 'KiB'],
     [10, 'KiB'],
-    [100, 'KiB'],
-    [1, 'MiB'],
-    [10, 'MiB'],
-    [100, 'MiB']
+    [100, 'KiB']
 ];
 
 const autoGenAutoUploadFileInfos: [number, Unit][] =
     process.env.AZLE_TEST_RUN_ON_RELEASE === 'true' ||
```

Comment:
Let's just use the regular test.yml variables, but prefixed with AZLE_. We should explicitly put in each case (release branch pr, main commit thing)